### DISCUSSION TITLE
Can sync to mobile

### DISCUSSION THREAD

**Discussion by Osirus2019** [remotely-save/remotely-save#267] [General]
How to use it on andriod tho, like i have connect dropbox to mobile obsidian, then i setup the same pwd as with the laptop one and i click sync, the message was success, but nothing was downloaded. So what is next, how do i solve this?

**Comment by Anyman552** [comment]
Have you created an empty Vault with the same name as your Vault? Worked for me.

**Comment by fyears** [comment]
empty vault with same name should work
